src/Plugin/CommerceDashboardItem/QueueStatus.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Drupal\acm\Plugin\CommerceDashboardItem;
+
+use Drupal\acm\CommerceDashboardItemBase;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Class ConnectorAccessiblityRequirement.
+ *
+ * @CommerceDashboardItem(
+ *   id = "queue_status",
+ *   title = @Translation("Queue status"),
+ *   weight = -100,
+ *   group = "tile",
+ * )
+ */
+class QueueStatus extends CommerceDashboardItemBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
```

Comment:
same here, need to implement the ContainerFactoryPluginInterface

Reply:
CommerceDashboardItemBase does implement that interface :)